apps/web/src/app/arrangementer/[slug]/[eventId]/page.tsx

```diff
@@ -194,4 +206,45 @@ const EventContent = ({ event, attendance, parentEvent, parentAttendance, punish
   )
 }
 
+export async function generateMetadata({ params }: { params: Promise<EventPageParams> }): Promise<Metadata> {
+  const { eventId } = await params
+
+  const detail = await server.event.find.query(eventId)
+
+  if (!detail) {
```

Comment:
I think you should be able to throw a redirect in here to a 404 page, but maybe this is preferable

Reply:
I think this is preferable